examples/launch_experi.py

```diff
@@ -0,0 +1,3 @@
+from experi.run import launch as launch_experi
```

Comment:
Since this function `launch` is designed to be used by importing, my opinion is that it should have a name which reflects that. That is, in this case the function name should be `launch_experi`, although having said that I think I like `experi_run` better. Having the function named `experi*` is easy to remember for tab completion.

Naming things is hard.

Reply:
Another way would be if the file structure in the package were slightly different you could have 

```python
import experi.launch

experi.launch("experiments.yml")
```
which is nice and explicit.

I don't know what you mean by tab completion, I tried Googling "python function asterisk tab completion" but nothing relevant came up. I suggest maybe this would be a case where simple but explicit naming might be better?